rhea/system/clock.py

```diff
@@ -46,6 +46,24 @@ def frequency(self, f):
     def period(self):
         return self._period
 
+    @property
+    def ticks(self):
+        """
+        Returns the number of simulation ticks that will elapse per clock tick.
+        """
+        return self._ticks
+
+    def delay(self, count=1):
```

Comment:
I was originally thinking of (and would have preferred to) implement delay() as yield delay(2*_hticks).   This may have better matched the myhdl delay(), but I was concerned about the fact that 2*_hticks can be off by one or two relative to simulation ticks due to their rounding.  This could be corrected via separate _hticks_high and _hticks_low values, that would of course slightly impact the duty cycle.

Anyways, after your suggestions it occurs to me that your good suggestions of 'posedges' and 'negedges' or 'edges'  really belong in Signal() and not Clock() - and may violate the more minimalist philosophy of such.   I'll remove delay(), but leave ticks in.